ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -65,6 +65,9 @@ public class ImagePickerSheetController: UIViewController, UITableViewDataSource
         }
     }
     
+    // Limit of selected images, if limit = 1 (previous will be unselected), if limit > 1 = you can't select new images.
+    public var limit = 0
```

Comment:
Could you rename this to 'maximumSelection'. I think that makes it clearer what it does. Could you also update the comment respectively.